hr_time/translations/de.csv

```diff
@@ -75,4 +75,20 @@ No time tracking,Keine Zeiterfassung
 Employee,Mitarbeiter*in
 Employee Checkin,Zeiterfassung
 "Documents and Events","Dokumente und Protokolle"
-Standard full-time 40 hours,Standard Vollzeit 40 Stunden
\ No newline at end of file
+Standard full-time 40 hours,Standard Vollzeit 40 Stunden
+Timestamp,Zeitstempel
+Time,Zeit
+Task Description,Aufgabenbeschreibung
+Task,Aufgabe
+Action,Aktion
+No employee ID found for the current user,Keine Mitarbeiter-ID für den aktuellen Benutzer gefunden
+Worklog,Arbeitsbericht
+Add Worklog,Arbeitsbericht hinzufügen
+Worklog added successfully,Arbeitsbericht erfolgreich hinzugefügt
+Worklog created successfully,Arbeitsbericht erfolgreich erstellt
+Failed to add Worklog,Konnte kein Arbeitsbericht hinzufügen
+Worklog description cannot be empty,Arbeitsbericht-Beschreibung kann nicht leer sein
```

Comment:
"Arbeitsbericht darf nicht leer sein"

(or: "Beschreibung des Arbeitsberichts darf nicht leer sein" - which would be a more concise translation, however, just telling the user that the "Worklog must not be empty" will probably be understandable? If not, feel free to go for the second translation)

Reply:
That is a good suggestion. I think "Worklog must not be empty" will be better.
I will add this instead Tobi.

Reply:
Hi Pratik,
sounds good. Also the DeepL translation sounds even better, so feel free to use "Die Aufgabenbeschreibung darf nicht leer sein" :)